tests/sentry/tasks/test_derive_code_mappings.py

```diff
@@ -81,8 +88,56 @@ def test_derive_code_mappings(self):
             derive_code_mappings(self.project.id, self.event_data)
             code_mapping = RepositoryProjectPathConfig.objects.all()[0]
             # ./app/foo.tsx -> foo.tsx -> static/app/foo.tsx
-            assert code_mapping.stack_root == "./app/"
-            assert code_mapping.source_root == "static/app/"
+            assert code_mapping.stack_root == "./"
+            assert code_mapping.source_root == "static/"
+            assert code_mapping.repository.name == repo_name
+
+    @responses.activate
+    @with_feature("organizations:derive-code-mappings")
+    def test_derive_code_mappings_not_static(self):
+        repo_name = "foo/bar"
+        self.create_integration(
+            organization=self.organization,
+            provider="github",
+            external_id=self.organization.id,
+            metadata={"domain_name": "github.com/Test-Org"},
+        )
+        with patch(
+            "sentry.integrations.github.client.GitHubClientMixin.get_trees_for_org"
+        ) as mock_get_trees_for_org:
+            mock_get_trees_for_org.return_value = {
+                repo_name: RepoTree(
+                    Repo(repo_name, "master"), ["app/utils/handleXhrErrorResponse.tsx"]
+                )
+            }
+            derive_code_mappings(self.project.id, self.event_data)
+            code_mapping = RepositoryProjectPathConfig.objects.all()[0]
+            # ./app/foo.tsx -> foo.tsx -> app/foo.tsx
```

Comment:
```suggestion
            # ./app/foo.tsx -> app/foo.tsx -> app/foo.tsx
```
The stack root drops "./", thus, `app/foo.tsx`